de.metas.adempiere.adempiere/client/src/main/java/de/metas/process/ui/ProcessParametersPanel.java

```diff
@@ -426,4 +427,19 @@ private void dynamicDisplay(final int fieldIndex)
 		// Ask the model to create the parameters
 		return model.createProcessInfoParameters();
 	}
+
+	/**
+	 * #782 Request focus on the first process parameter (if possible)
+	 */
+	public void focustFirstParameter()
+	{
+		if(fieldEditors.size() < 1)
+		{
+			// there are no parameters in this process. Nothing to focus
+			return;
+		}
+		final VEditor editor = fieldEditors.get(0);
+		getComponent(editor).requestFocusInWindow();
```

Comment:
what happens if requestFocusInWindow() returns false.
Instead, i would suggest to iterate the editors, requestFOcusInWindow() and stop on first one which returns true.